### ISSUE TITLE
Autopatch keeps linked nodes keyed

### ISSUE THREAD

**Issue by eliggett** [AllStarLink/app_rpt#398]
Greetings, 

With the latest updates to app_rpt and dahdi-linux, autopatch on my simplex node is working smoothly with a functioning vox. 

However, if any nodes are linked during an autopatch call, the linked nodes remain  keyed for the entirety of the autopatch call. This is not only inconvenient, but it can tie up resources significantly depending on what you have connected, making it difficult for others (with half-duplex nodes) to disconnect from your active node with the autopatch call. 

It would be great if the same PTT action created by the autopatch vox could be sent to remote nodes so that they are not keyed the entire time. 

Thanks for your consideration, 

--E
de W6EL

**Comment by jxmx** [comment]
@eliggett - Does the DAHDI 3.4.0 modules fix this issue for you?

**Comment by eliggett** [comment]
Negative, still keying the other nodes full time.

**Comment by jxmx** [comment]
@mkmer - Is this still needed? If so, can the actual enhancement be documented?

**Comment by mkmer** [comment]
I don't think we can change how this works.  If there is a way, it's not obvious to me.

**Comment by jxmx** [comment]
Closing.
